src/lib/containers/table/SynapseTable.tsx

```diff
@@ -820,28 +791,27 @@ export default class SynapseTable extends React.Component<
                   </a>
                 )}
 
-                {!isCountColumn && (
+                {!isCountColumn && showCardLabel && (
+                  <SynapseCardLabel
+                    value={columnValue}
+                    columnName={columnName}
+                    selectColumns={selectColumns}
+                    columnModels={columnModels}
+                    isHeader={false}
+                    labelLink={columnLinkConfig}
+                    rowData={row.values}
+                  />
+                )}
```

Comment:
I pulled this out of the SynapseTableCell component. My reasoning for this is that I'm guessing that using a SynapseCardLabel is exclusively for Portals use cases (which SynapseTable is for). I'd like to reuse SynapseTableCell for other purposes, like the new Dataset editor.

Many of these components could use better names.